src/js/client/editprefset-dialogs-widget.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Dialog boxes for the DevPTT.
+ */
+"use strict";
+
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.devpmt");
+fluid.registerNamespace("gpii.devpmt.dialogs");
+
+/**
+ * Add NP Set Context Dialog
+ */
+fluid.defaults("gpii.devpmt.dialogs.addContextDialog", {
+    gradeNames: ["gpii.devpmt.dialogs.confirmDialog", "gpii.binder.bindOnCreate", "gpii.binder.bindOnDomChange"],
```

Comment:
Search for `bindOnDomChange` throughout and review carefully.  This also inherits `templateAware` from a base grade and should not need to have `gpii.binder.bindOnDomChange` as an additional grade.

Reply:
All instances of this should be fixed up now.